docs/pages/application-access/guides/azure.mdx

```diff
@@ -0,0 +1,554 @@
+approach is to define a user's permitted Azure identities by configuring each
+user separately.
+
+To do so, modify the `azure-cli-access` role you defined above to include the
+following:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: azure-cli-access
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    azure_identities:
+      - '{{internal.azure_identities}}'
+```
+
+In this case, when a user authenticates to the Azure CLI via Teleport, the
+Teleport Auth Service populates the `{{internal.azure_identities}}` template
+variable with any Azure identities you have assigned to the user.
+
+To assign Azure identities to a user, run a `tctl users update` command similar
+to the following:
+
+```code
+$ TELEPORT_USER=myuser;
+$ ID1=one_id;
+$ ID2=another_id;
+$ SUBSCRIPTION_ID=00000000-0000-0000-0000-000000000000;
+$ RESOURCE_GROUP="my-resource-group";
+$ tctl users update ${TELEPORT_USER?} --set-azure-identities \
+ `/subscriptions/${SUBSCRIPTION_ID?}/resourceGroups/${MY_RESOURCE_GROUP?}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/${ID1?},\
+ `/subscriptions/${SUBSCRIPTION_ID?}/resourceGroups/${MY_RESOURCE_GROUP?}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/${ID2?}
+```
+
+This command uses the `--set-azure-identities` flag to add Azure identities to a
+user. The value of this flag is a comma-separated list of Azure identity URIs.
+
+You must attach all identities you would like users to access to the Azure VM
+running the Application Service so the Application Service can make requests to
+the Azure API using those identities.
+
+</Details>
+
+## Step 4/4. Use the Azure CLI with Teleport
+
+Now that you have authorized your Teleport user to assume the `teleport-azure`
+identity, you can use Teleport to authenticate to the Azure API and execute
+commands against it via the `az` CLI.
+
+### List your Azure CLI application
+
+Verify that your Teleport user can see the `azure-cli` application you
+registered earlier:
+
+```code
+$ tsh apps ls
+Application Description Type Public Address                 Labels
+----------- ----------- ---- ------------------------------ -------------------
+azure-cli               HTTP azure-cli.teleport.example.com teleport.dev/origin
+```
+
+### Log in to use the Azure CLI
+
+Log in to the application, specifying that you would like to assume the
+`teleport-azure` identity:
+
+```code
+$ tsh app login azure-cli --azure-identity teleport-azure
+```
+
+This command validates the value of the `--azure-identity` flag against the ones
+the user is authorized to assume. The value of the flag can either be the full
+URI of the identity (e.g., the URI you copied earlier in this guide) or the name
+of the identity, e.g., `teleport-azure`. 
+
+A user can leave the `--azure-identity` flag blank if they are only authorized
+to access a single Azure identity, but otherwise an empty `--azure-identity`
+will result in an error.
```

Comment:
Would they live it blank or not include the flag?

Reply:
If they have access to exactly one identity, the flag can be omitted. Otherwise, you need to specify the identity. Perhaps this paragraph could be rephrased to educate about the reason the flag is sometimes required and sometimes not.

Reply:
👍